tools/system_libs.py

```diff
@@ -1382,7 +1382,7 @@ def can_build(self):
 class libwasmfs(MTLibrary, DebugLibrary):
   name = 'libwasmfs'
 
-  cflags = ['-fno-exceptions', '-std=c++17']
```

Comment:
I think it would have been better to not land this change here, to keep this PR focused on just being a bugfix. But no big deal. I'll close my PR then.

Separately, for the record, I do think a getter/setter approach would be best overall...